modules/tinymce/src/plugins/table/test/ts/browser/command/TableRowHeaderCommandSectionCellsTest.ts

```diff
@@ -0,0 +1,517 @@
+/* eslint-disable mocha/no-exclusive-tests */
+import { context, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/table/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+import { clickOnButton, makeCell } from '../../module/test/TableModifiersTestUtils';
+
+describe('browser.tinymce.plugins.table.command.TableRowHeaderCommandSectionCellsTest', () => {
```

Comment:
Are these three test files identical except for the `table_header_type` setting in the config? 

It should be one test with something like this:

```
Arr.each([ 'sectionCells', 'cells', 'section' ], (headerType) => {
  context(`table_header_type: ${headerType}`, () => {
    const hook = TinyHooks.bddSetupLight<Editor>({
      plugins: 'table',
      indent: false,
      toolbar: 'tablerowheader',
      table_header_type: headerType,
      base_url: '/project/tinymce/js/tinymce',
    }, [ Plugin, Theme ], true);
  });
})
```

- [ ] Fix test duplication

Reply:
I've just noticed the slight differences in behaviour, so the files aren't identical, but this still should be improved to reduce the significant duplication.

Reply:
`table_header_type` is extensively covered in `SwitchTableSectionTest.ts`, `TableSectionApiTest.ts`, and others. Are these tests covering anything new?

Reply:
I don't see any really easy way to reduce duplication, unfortunately. The setups are different for the different tests, and the outcome is further different depending on the setting. It might be possible to make some array-arrangement, but I don't think it'd be any easier to read.

As for testing it, this does go through the query `mceTableRowType` and the command `mceTableRowType`, so it's not simply testing the `table_header_type`. Deleting the different variants aside, this would still test other things regarding selection and switching. We can probably see if it tests something the others does not to see if we still want the variants.

Name can probably be changed though.

Reply:
`SwitchTableSectionTest` does a lot of tests for switching, I suspect the internal logic for the commands is quite well tested, so we likely only need your new `TableRowHeaderUiTest.ts` to test the new buttons work